### PR TITLE
Rename the special lock expression "itself" to "<self>"

### PR DESCRIPTION
**Pull request by jthaine**

### CODE REVIEW

File: checker/tests/lock/FlowExpressions.java [hunk 1]
```diff
@@ -24,17 +24,5 @@ public void method() {
         synchronized(getm()) {
             getm().field = new Object();
         }
-
```
Comment (mernst) [inline checker/tests/lock/FlowExpressions.java]: Why is this block removed?

Comment (jthaine) [inline checker/tests/lock/FlowExpressions.java]: It was testing a situation where "itself" could refer to either a variable (within the block) or "<self>" (outside of this block).

**Comment by mernst** [comment]
Looks good.  I had one question.  Feel free to merge when that is resolved and the tests pass.

File: checker/tests/lock/FlowExpressions.java (hunk 1 above)
Comment (mernst) [inline checker/tests/lock/FlowExpressions.java]: OK, sounds good.  Thanks.

**Pull request merged**